### ISSUE TITLE
functionality for the Meross MSG 100 garage door opener.

### ISSUE THREAD

**Issue by Vati80** [Robdel12/homebridge-meross-plug#9]
Hi,

can you please include the functionality for the MSG 100 garage door opener. That would be really great.
The Homebridge plugin can be installed and he also displays the device as a "switch" in the home app. In the home app, it can also be "switch" (icon lights up and darkens) but the switch, or the relay in the switch does not respond/ click.

many thanks Vati

**Comment by Aamerikan** [comment]
Yeah for real, This would be by far the best option for a Homekit Garage door opener. I would legit pay you to create a plugin for the MSG100 Garage door opener.
